resources/suite-runner.mjs

```diff
@@ -100,5 +100,6 @@ class RemoteSuiteRunner extends SuiteRunner {}
 export const SUITE_RUNNER_LOOKUP = {
     __proto__: null,
     default: SuiteRunner,
+    async: SuiteRunner,
```

Comment:
no need to create a new SuiteRunner, since only the TestRunner is different and that's dynamically selected.